data/json/npcs/Lighthouse_Family/NPC_lighthouse_man.json

```diff
@@ -0,0 +1,464 @@
+      "success_lie": "-",
+      "failure": "-"
+    }
+  },
+  {
+    "id": "MISSION_lighthouse_man_2",
+    "type": "mission_definition",
+    "has_generic_rewards": false,
+    "name": { "str": "Clean lighthouse" },
+    "description": "Clean lighthouse from <zombies> with Mikhail.",
+    "goal": "MGOAL_CONDITION",
+    "goal_condition": { "npc_has_var": "lighthouse_man", "type": "count", "context": "cleaning", "value": "6" },
+    "difficulty": 0,
+    "value": 5000,
+    "start": {
+      "effect": [
+        "follow_only",
+        { "npc_first_topic": "TALK_lighthouse_man_cleaning" },
+        { "npc_add_var": "lighthouse_man", "type": "count", "context": "cleaning", "value": "0" }
+      ]
+    },
+    "end": {
+      "opinion": { "trust": 3, "value": 3 },
+      "effect": [
+        "stop_following",
+        { "npc_first_topic": "TALK_lighthouse_man_1" },
+        { "npc_lose_var": "lighthouse_man", "type": "count", "context": "cleaning" }
+      ]
+    },
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_lighthouse_man_3",
+    "dialogue": {
+      "describe": "-",
+      "offer": "So, we found the lighthouse.  Now, I need to clean it.  Want to help me with it?",
+      "accepted": "Great.  Let's do it together.  I'll be right behind you.  We will clean floor by floor.  If you decide that the floor is clean, tell me about it, and we will move on to the next, and so on to the very top.",
+      "rejected": "Well, it's your choice.",
+      "advice": "Be <very> careful.  These <name_b>s are most dangerous in tight spaces.",
+      "inquire": "<its_safe>",
+      "success": "Great, <very> great.  All that remains is to wash all the floors…",
+      "success_lie": "-",
+      "failure": "-"
+    }
+  },
+  {
+    "id": "MISSION_lighthouse_man_3",
+    "type": "mission_definition",
+    "name": { "str": "Fisherman's family" },
+    "description": "Mikhail asked you to bring back his family.",
+    "goal": "MGOAL_ASSASSINATE",
+    "difficulty": 0,
+    "value": 25000,
+    "start": {
+      "effect": { "npc_first_topic": "TALK_lighthouse_man_family" },
+      "assign_mission_target": { "om_terrain": "house_04_basement", "reveal_radius": 2 },
+      "update_mapgen": {
+        "place_npcs": [
+          { "class": "NPC_lighthouse_woman", "x": 4, "y": 8, "target": true },
+          { "class": "NPC_lighthouse_girl", "x": 3, "y": 8 }
+        ]
+      }
+    },
+    "end": {
+      "opinion": { "trust": 6, "value": 6 },
+      "effect": { "u_add_var": "lighthouse_family_safe", "type": "flag", "context": "help", "value": "yes" }
+    },
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_lighthouse_man_4",
+    "dialogue": {
+      "describe": "-",
+      "offer": "We found The lighthouse and cleared it of <zombies>.  Now, I need to bring my family here.  May I ask you for a favor?  Since you bring me here, can you go back, find my family and bring them here?",
```

Comment:
```suggestion
      "offer": "We found the lighthouse and cleared it of <zombies>.  Now, I need to bring my family here.  May I ask you for a favor?  Since you bring me here, can you go back, find my family and bring them here?",
```